tools/figma-inspector/src/utils/utils.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: MIT
+import { dispatchTS } from "./bolt-utils.js";
+
+export function writeTextToClipboard(str: string) {
```

Comment:
This function looks asynchronous as it returns a Promise, but AFAICS the executor is 100% synchronous. I think you can simplify the functions to not return a promise - especially given that the caller in line 31 doesn't care about the return value anyway :)